R/pipeline.R

```diff
@@ -0,0 +1,224 @@
+#' Run an Rt Estimation Model Pipeline
+#'
+#' This function runs a complete pipeline for fitting an Rt estimation model,
+#' using the `EpiNow2` model, based on a configuration file. The pipeline
+#' processes the model, logs its progress, and handles errors by logging
+#' warnings and setting the pipeline status. Output and logs are written to
+#' the specified directories. Additionally, support for uploading logs and
+#' outputs to a blob storage container is planned.
+#'
+#' @param config_path A string specifying the file path to the JSON
+#' configuration file. The configuration file should include at least `job_id`
+#' and `task_id` to manage the logging and task metadata.
```

Comment:
If these two params are not sufficient to support the model run, I'd leave off this detail and just link to cofig documentation later.

Reply:
https://github.com/CDCgov/cfa-epinow2-pipeline/pull/79/commits/f56e176d26e5c1df2f21d063545464158dedc4bd